modules/rn_3_90.adoc

```diff
@@ -92,16 +92,40 @@ For more information, see link:https://access.redhat.com/documentation/en-us/red
 
 ** **FEATURE_SECURITY_SCANNER_NOTIFY_ON_NEW_INDEX**: Whether to allow sending notifications about vulnerabilities for new pushes.
 +
-*Default**: `True`
+**Default**: `True`
 +
 For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/configure_red_hat_quay/index#config-fields-scanner[Security scanner configuration fields].
 
+* The following configuration field has been added to configure whether {productname} automatically removes old persistent volume claims (PVCs) when upgrading from version 3.8 -> 3.9:
+
+** **POSTGRES_UPGRADE_RETAIN_BACKUP**: When set to `True`, persistent volume claims from PostgreSQL 10 are backed up. 
++
+**Default**: `False`
+
 [id="quay-operator-updates"]
 == {productname} Operator
 
 The following updates have been made to the {productname} Operator:
 
-*
+* Currently, the {productname} Operator and Clair use PostgreSQL 10. PostgreSQL 10 will have its final release in May, 2024. Afterwards, it will be unsupported.
++
+With this release, if your database is managed by the {productnmame} Operator, updating from {productname} 3.8 -> 3.9 automatically handles upgrading PostgreSQL 10 to PostgreSQL 13. This automatic procedure works by first upgrading your PostgreSQL 10 to PostgreSQL 12, and then from PostgreSQL 12 to PostgreSQL 13.
```

Comment:
@dmage after refactor the upgrade process, do we still need interim `PostgreSQL 12` stage? looks it upgrades from PostgreSQL 10 to PostgreSQL 13 directly